submissions/ayotolaTehingbola/css/stylesheet.css

```diff
@@ -0,0 +1,221 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+img {
+  max-width: 100%;
+  display: block;
+}
+
+a {
+  text-decoration: none;
+  color: white;
+}
+
+a:hover {
+  color: teal;
+}
+
+a:active {
+  text-decoration: underline;
+}
+
+.header {
+  background-color: black;
+  color: white;
+  padding: 10px;
+  margin: auto;
+  overflow: hidden;
+}
+
+.navbar {
+  text-align: right;
+  text-decoration: none;
+  color: white;
+  margin-right: 40px;
+}
+
+.navbar-list {
+  list-style-type: none;
+}
+
+.navbar-list li {
+  display: inline;
+  margin: 15px;
+  text-decoration: none;
+}
+
+.container {
+  background-color: white;
+  color: black;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  font-size: 14px;
+  font-weight: normal;
+  line-height: 1.6em;
+}
+
+.inner-container {
+  margin-top: 30px;
+  margin-right: 30px;
+  margin-left: 30px;
+  margin-bottom: 30px;
+  overflow: hidden;
+}
+
+.my-form {
+  display: block;
+}
+
+#existing-account-signin {
+  color: teal;
+  text-decoration: underline;
+}
+
+#main-block {
+  box-sizing: border-box;
+  float: left;
+  padding: 15px;
+  margin-top: 150px;
+  text-align: center;
+  width: 50%;
+}
+
+#side-bar {
+  float: right;
+  width: 50%;
+  padding: 25px;
+  columns: white;
+  border: 0.3px black solid;
+  border-radius: 10px;
+  background-color: whitesmoke;
+  text-align: justify;
+}
+
+input,
+textarea {
+  width: 100%;
+  height: 50px;
+}
+
+#firstname {
+  border-radius: 0.4em;
+  padding: 5px;
+  border: azure;
+}
+
+#lastname {
+  border-radius: 0.4em;
+  padding: 5px;
+  border: azure;
+}
+
+#email {
+  border-radius: 0.4em;
+  padding: 5px;
+  border: azure;
+}
+
+#address {
+  border-radius: 0.4em;
+  padding: 5px;
+  border: azure;
+}
+
+#password {
+  border-radius: 0.4em;
+  padding: 5px;
+  border: azure;
+}
+
+#cpassword {
+  border-radius: 0.4em;
+  padding: 5px;
+  border: azure;
+}
```

Comment:
I notice that you have the same CSS properties from #firstname to #cpassword. You can use one CSS selector to access these 6 IDs (from #firstname to #cpassword) instead of creating them multiple times. Better still, create a class with the properties and add the class to those elements.